examples/renameimage.py

```diff
@@ -38,44 +97,100 @@
 
     glancesync = GlanceSync()
     glancesync.init_logs()
+    master = glancesync.master_region
+
+    master_images = glancesync.get_images_region(master)
+
+    old_image = None
+    new_image = None
+    for image in master_images:
+        if image.name == sys.argv[1]:
+            if old_image:
+                msg = 'There are more than an image with the old name in the' \
+                      ' master region'
+                logging.error(msg)
+                sys.exit(-1)
+            else:
+                old_image = image
+        elif image.name == sys.argv[2]:
+            if new_image:
+                msg = 'There are more than an image with the new name in the' \
+                      ' master region'
+                logging.error(msg)
+                sys.exit(-1)
```

Comment:
logging.error by default uses the root handler, that prints a message to sys.stderr.